src/integrationTest/java/uk/gov/hmcts/reform/sscs/config/SwaggerPublisherTest.java

```diff
@@ -31,15 +24,15 @@
     @Test
     @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
     void generateDocs() throws Exception {
-        byte[] specs = mvc.perform(get("/v3/api-docs"))
+        /*byte[] specs = mvc.perform(get("/v3/api-docs"))
```

Comment:
Is this still needed ? If not, delete it.

Reply:
Yes it needed and It will cover in swagger api change